templates/rustdoc/topbar.html

```diff
@@ -226,7 +226,26 @@
                 </li>
             {%- endfor -%}
         </ul>
-    </li>
+    </li>{#
+    Display the features available in current build
+    #}
+    {% if krate.features %}
```

Comment:
You need `{%-` to remove the whitespace (which @Nemo157 said broke things for some reason).
```suggestion
    {%- if krate.features -%}
```

Reply:
Not _breakage_, just an extra space gets rendered making the spacing between elements in the header inconsistent :grin: